packages/iris-grid/src/sidebar/visibility-ordering-builder/VisibilityOrderingBuilder.tsx

```diff
@@ -168,6 +179,50 @@ class VisibilityOrderingBuilder extends Component<
         this.list?.querySelectorAll('.item-wrapper')[visibleIndexToFocus];
       columnItemToFocus?.scrollIntoView({ block: 'center' });
     }
+
+    this.setState({
+      prevQueriedColumns: columnsMatch,
+      queriedColumnIndex: undefined,
+    });
+  }
+
+  /**
+   * Change the selected column to either the next or previous column
+   *
+   * @param direction The direction to move the selection
+   */
+
+  changeSelectedColumn(direction: 'forward' | 'back'): void {
```

Comment:
i feel this could use a better name but not too sure what. This does change the selected column as the name implies but only from the previously queried list